inspirehep/modules/literaturesuggest/tasks.py

```diff
@@ -153,7 +153,11 @@ def _is_arxiv_url(url):
         else form_fields.get('language')
     builder.add_language(language=language)
 
-    builder.add_title_translation(title=form_fields.get('title_translation'))
+    if form_fields.get('title_translation'):
+        builder.add_title_translation(
+            title=form_fields['title_translation'],
+            language='en',
+        )
```

Comment:
`formdata_to_model` here was calling `add_title_translation` without `language`, which is required by the schema, so it should be a required parameter of the builder method: https://github.com/inspirehep/inspire-schemas/pull/288